Src/BootCamp.Chapter/TransactionProcessors/FullProcessor.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using BootCamp.Chapter.Objects;
+
+namespace BootCamp.Chapter.TransactionProcessors
+{
+    public static class FullProcessor
+    {
+        public static void ExportFullSummary(IEnumerable<Transaction> transactions)
+        {
+            var transactionDatabase = transactions
+                .GroupBy(n => n.Shop)
+                .Select(n => n.ToList());
+
+            foreach (var transaction in transactionDatabase)
+            {
+                GetShopSummary(transaction);
+            }
+        }
+        
+        private static void GetShopSummary(List<Transaction> shop)
+        {
+            var storeName = shop.FirstOrDefault()?.Shop;
+            var summary = new StringBuilder();
+            summary.AppendLine("City, Street, Item, DateTime, Price");
+            
+            foreach (var transaction in shop)
```

Comment:
If logic can be in lower levels, it should. What is the lowest level in this case? It's the transaction. Let transaction print its' own info.

Then higher-level code will have less burden of low-level concerns.